views/editor/nav-options.html

```diff
@@ -112,14 +112,14 @@ <h2 class="snippets-menu-title">{{ gettext("snippetMenuTitle") }}</h2>
         </div>
         <div id="editor-pane-nav-options-menu" class="hide">
           <ul>
-            <li id="editor-option-textsize">
+            <li title="{{ gettext("Change the editor font size") }}" id="editor-option-textsize">
```

Comment:
We need to add a new string to the `server.properties` file, which each locale can translate into the appropriate language.  You can add something like `navTextSizeTitle` with this string in https://github.com/mozilla/thimble.mozilla.org/blob/224ecbe154f5ab855e500db36fbecc003d31c272/locales/en-US/server.properties#L383-L414